src/index.ts

```diff
@@ -26,7 +26,7 @@ const defaultExtensions = [
 
 export const interfaceVersion = 2
 
-export interface TsResolverOptions {
+type TsResolverOptions = SyncOpts & {
```

Comment:
```suggestion
export interface TsResolverOptions extends SyncOpts {
```

Reply:
Because `SyncOpts` is a `type` and not an `interface`, `TSResolverOptions` can not extend it. Can see issue arise in editor: 
![image](https://user-images.githubusercontent.com/16494982/127614156-5ee68258-302e-4f1e-b43b-c6e44edc0a33.png)

Reply:
OK, but please `export` it at least.

Reply:
silly me -- resolved